doc/schema/schema.md

```diff
@@ -79,7 +90,7 @@ Observed data on which the `posterior` is conditional. It should only contain da
 Model constants, data included in the model which is not modeled as a random variable. It should be the data used to generate the `posterior` and `posterior_predictive` samples.
 
 ### `prior`
-Samples from the prior distribution p(theta). Samples need not match `posterior` samples. However, this group will still follow the convention on `chain` and `draw` as first dimensions. Each variable should have a counterpart in `posterior`.
+Samples from the prior distribution p(theta). Samples need not match `posterior` samples. However, this group will still follow the convention on `chain` and `draw` as first dimensions. It should have matching variables with the `posterior` group.
 
 ### `sample_stats_prior`
 Information and diagnostics for each `prior` sample, provided by the inference backend. It may vary depending on the algorithm used by the backend. Variable names follow the same convention defined in [`sample_stats`](#sample_stats).
```

Comment:
I don't think "for each `prior` sample" is correct, is it?  Shouldn't it be 

"Information and diagnostics for the samples in the `prior` group, provided by the inference backend."?